generate-commitish.sh

```diff
@@ -5,4 +5,5 @@ VERSIONTAG="$(git describe --tags --abbrev=0)"
 BRANCH="$(git rev-parse --abbrev-ref HEAD)"
 COMMITSHA="$(git log -1 --format=%h)"
 
-echo "$VERSIONTAG ($BRANCH: $COMMITSHA)"
\ No newline at end of file
+export REACT_APP_VERSION="$VERSIONTAG ($BRANCH: $COMMITSHA)"
+echo "$VERSIONTAG ($BRANCH: $COMMITSHA)"
```

Comment:
Prefer `echo $REACT_APP_VERSION`, if we are keeping this side effect inside the script (see comment above).